modules/ROOT/pages/portal-vanity-domain.adoc

```diff
@@ -119,6 +119,32 @@ NOTE: Your server must run using `https`.
 
 If set up correctly, you can access and navigate to `+https://developers.robocop.com/path/of/justice/+` exactly as if you were at `+https://anypoint.mulesoft.com/exchange/portals/robo-cop/+`.
 
+== Set up Cookie Consent Management for your domain
+
+A cookie consent managment solution is provided out of the box for Exchange public portals that don't use vanity domains.
```

Comment:
Spell check: management
Also, we should perhaps add a link to what is cookie consent management. That document does not exist yet.

Reply:
Hi @jyotirmi, I do not know if there will be any external page that we can add as a reference or if one will be created soon within the mulesoft docs, in any case I could add an example image of the consent manager cookie that we currently have in exchange, which is a modal like this:

![image](https://user-images.githubusercontent.com/11414027/111158545-e7693900-8576-11eb-898e-0286788d8c17.png)

This is not an explanation but the customer could get an idea of what it is 😛 .